db/migrate/20250407150531_add_order_to_relationships.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrderToRelationships < ActiveRecord::Migration[7.0]
+  def change
+    add_column :core_data_connector_relationships, :order, :integer
```

Comment:
Should we default this to "0" to avoid `nil` order values?

Reply:
Now that I'm working on the `core-data-cloud` side, I'm wondering if we shouldn't use a default value. If I have a list of related records that all have an `order` value of 0, and I set one of them to 1, it will be last in the list. Getting one to appear on top (e.g. if I want it to be featured on the CDP detail page) would require setting an order value for every related record, since 0 will always come first.

When `order` defaults to `nil`, the `nil` values are sorted _below_ the non-`nil` values. So if I have a bunch of relations with `order: nil` and set one of them to 1, it will immediately be first on the list.

Reply:
This makes sense to me 👍